### DISCUSSION TITLE
Build Sphinx docs for API

### DISCUSSION THREAD

**Discussion by fepegar** [microsoft/InnerEye-DeepLearning#663] [Q&A]
Is there a known way to build the Sphinx docs including the API for the package? It would be nice to see all those docstrings nicely rendered on a browser.

**Comment by ant0nsc** [comment]
yes! you go into the `docs` folder, and run `make html`.

This is mentioned in `contributing.md` and `developers.md`.

**Reply by fepegar** [reply]
I think you're referring to `hi-ml`.
